client/src/components/DisplayTableUtils.scss

```diff
@@ -0,0 +1,30 @@
+@import '../common_css/common-variables';
+
+.heading-container {
```

Comment:
Class names in this file are fairly generic, could they reference display table in some way?

Reply:
I don't think so, I changed them up however